Makefile

```diff
@@ -66,7 +66,7 @@ endif
 	rm -rf $(tmpdir)
 
 .PHONY: dist
-dist: lint test clean build package
+dist: lint test-unit clean package
```

Comment:
does the 'package' run the 'build' as well? just wondering if we should add 'build' back in to this line